hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exception/UnknownHederaFunctionality.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2020-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.hapi.utils.exception;
+
```

Comment:
Same with this. We can move this to hapi-utils. It currently exists `hedera-app`.

Reply:
I updated the change you asked. Please re-check. Thanks.